py-polars/tests/unit/io/test_delta.py

```diff
@@ -487,3 +487,20 @@ def test_scan_delta_DT_input(delta_table_path: Path) -> None:
 
     expected = pl.DataFrame({"name": ["Joey", "Ivan"], "age": [14, 32]})
     assert_frame_equal(expected, ldf.collect(), check_dtypes=False)
+
+
+@pytest.mark.skip(
+    reason="""\
+Upstream bug writing empty tables \
+"_internal.DeltaError: Generic error: No data source supplied to write command"
+Note this works if we downgrade to deltalake==0.18.2
+"""
+)
+@pytest.mark.write_disk
+def test_read_delta_empty(tmp_path: Path) -> None:
+    tmp_path.mkdir(exist_ok=True)
+    path = str(tmp_path)
+    df = pl.DataFrame({}, [("p", pl.Int64)])
+    df.write_delta(path)
```

Comment:
@ion-elgreco , can you check this one? I get an error writing with this on deltalake 0.21.0

Reply:
You should create empty tables with Deltatable.create()

Reply:
I see - thanks for the tip!